lib/saml_idp_constants.rb

```diff
@@ -5,11 +5,6 @@ module Constants
       LOA1_AUTHNCONTEXT_CLASSREF = 'http://idmanagement.gov/ns/assurance/loa/1'.freeze
       LOA3_AUTHNCONTEXT_CLASSREF = 'http://idmanagement.gov/ns/assurance/loa/3'.freeze
       REQUESTED_ATTRIBUTES_CLASSREF = 'http://idmanagement.gov/ns/requested_attributes?ReqAttr='.freeze
-
```

Comment:
if we can keep the namespace, maybe a comment here pointing at `config/initializers/saml_idp.rb` ?
I get there is probably a load order issue with doing the Figaro check in `lib/*` but this really is a constant, imo.